Corrections for Choreo Connect documentation (#4961)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md

### 3. Adapter

Router and Enforcer have a data plane API that can be used to dynamically configure them.

> When an API needs to be deployed on the Choreo Connect we need to translate that API definition to the format understood by the Router (convert the API into routes) and Enforcer.

Reviewer: ```suggestion
When an API needs to be deployed in Choreo Connect, you need to translate that API definition to the format understood by the Router (convert the API into routes) and Enforcer.
```
